README.md

```diff
@@ -33,6 +33,38 @@ The recommended import order is as follows:
 - `animations`
 - `player`
 
+## Sending Segment Events
```

Comment:
Thanks for adding this readme. It is especially useful as the webflow code cannot be committed